webapp/src/client.ts

```diff
@@ -155,17 +159,8 @@ export default class CallsClient extends EventEmitter {
 
         ws.on('join', async () => {
             console.log('join ack received, initializing connection');
-            let config;
-            try {
-                const resp = await axios.get(`${getPluginPath()}/config`);
-                config = resp.data;
-            } catch (err) {
-                console.log(err);
-                this.ws?.close();
-                return;
-            }
-
-            const iceServers = config.ICEServers?.length > 0 ? [{urls: config.ICEServers}] : [];
+            const configServers = getIceServers(this.getState());
```

Comment:
Is there a specific reason we want to be requesting fresh iceServers on `join`, or was that just a way to get the data? We can still request fresh data on `join` if we want to.

Reply:
I think it was just about needing them there which automatically supported the case of changes at runtime but it's not a big requirement as it's mostly static information, configured once and left there.